lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -383,6 +383,12 @@ static void addClosureSpecializePassPipeline(SILPassPipelinePlan &P) {
   P.addDeadFunctionElimination();
   P.addDeadObjectElimination();
 
+  // These few passes are needed to cleanup between loop unrolling and GlobalOpt.
```

Comment:
How much do gain for adding these passes? Is it worth the extra compile time?

Reply:
The passes are needed to completely optimize away the static small string constants.
I can add that comment in a follow up commit.

About compile time: I didn't see any noticeable impact

Reply:
Sounds good!